[2024-05-13T03:58:43.793+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-13T03:58:43.807+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet scheduled__2024-01-30T00:00:00+00:00 [queued]>
[2024-05-13T03:58:43.814+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet scheduled__2024-01-30T00:00:00+00:00 [queued]>
[2024-05-13T03:58:43.814+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-13T03:58:43.821+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): greet> on 2024-01-30 00:00:00+00:00
[2024-05-13T03:58:43.824+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=5044) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-13T03:58:43.824+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v01', 'greet', 'scheduled__2024-01-30T00:00:00+00:00', '--job-id', '882', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmppzveck2e']
[2024-05-13T03:58:43.826+0000] {standard_task_runner.py:63} INFO - Started process 5049 to run task
[2024-05-13T03:58:43.826+0000] {standard_task_runner.py:91} INFO - Job 882: Subtask greet
[2024-05-13T03:58:43.849+0000] {task_command.py:426} INFO - Running <TaskInstance: our_dag_with_python_operator_v01.greet scheduled__2024-01-30T00:00:00+00:00 [running]> on host f1ddc6e75993
[2024-05-13T03:58:43.890+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mani' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v01' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2024-01-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-30T00:00:00+00:00'
[2024-05-13T03:58:43.891+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-13T03:58:43.896+0000] {logging_mixin.py:188} INFO - Hello World
[2024-05-13T03:58:43.897+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-13T03:58:43.898+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-13T03:58:43.903+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v01, task_id=greet, run_id=scheduled__2024-01-30T00:00:00+00:00, execution_date=20240130T000000, start_date=20240513T035843, end_date=20240513T035843
[2024-05-13T03:58:43.925+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-13T03:58:43.934+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-13T03:58:43.935+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
